Report.java

```diff
@@ -202,4 +205,16 @@ private void printSpriteCounts(PrintWriter printW, int i)
             printW.println();
         }
     }   
+
+    /**
+     * Method to get the current datetime for report.
+     *
+     * @return formatted datetime
+     */
+    public String getReportDateTime()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
```

Comment:
I think it's standard to use all 4 digits for the year, so probably best to leave that. Yes, we'll definitely need to update the READ.ME...maybe just say Report-YYYYMMDDHHSS.txt?